backend/src/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const ERROR_CODES = {
```

Comment:
I think the better location for this file could be:
`backend/src/api/utils/error-codes.ts`

Reply:
Ok, I will refractor it to the path and change file name.

> I think the better location for this file could be: `backend/src/api/utils/error-codes.ts`



> I think the better location for this file could be: `backend/src/api/utils/error-codes.ts`